### PR TITLE
feat: update service config template

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: frontend/constants/serviceTemplates.ts [hunk 1]
```diff
@@ -157,31 +157,61 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
   //   },
   {
     agentType: AgentType.Memeooorr,
-    name: '', // TODO: fix value
+    name: 'Memeooorr',
     hash: '', // TODO: fix value
```
Comment (Tanya-atatakai) [inline frontend/constants/serviceTemplates.ts:161]: @OjusWiZard please let me know if you know what should be here 🙏

File: frontend/constants/serviceTemplates.ts [hunk 2]
```diff
@@ -157,31 +157,61 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
         fund_requirements: {
-          agent: 100000000000000000, // TODO: fix value
-          safe: 5000000000000000000, // TODO: fix value
+          agent: 1000000000000000, // 0.001
+          safe: 2000000000000000, // 0.002
         },
       },
     },
-    env_variables: {}, // TODO: fix values
+    env_variables: {
```
Comment (Tanya-atatakai) [inline frontend/constants/serviceTemplates.ts:183]: we're gonna pass twitter variables (email, username, password, gemini api key and persona description) like this, let me know if that would work

File: frontend/constants/serviceTemplates.ts [hunk 3]
```diff
@@ -157,31 +157,61 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
+      TWIKIT_USERNAME: {
+        name: 'Twitter username',
+        description: '',
+        value: '',
+        provision_type: EnvProvisionType.USER,
```
Comment (Tanya-atatakai) [inline frontend/constants/serviceTemplates.ts:188]: unsure how this works, would appreciate if @truemiller or @OjusWiZard some of you confirm if all these should have `provision_type: "user"`

**Review by truemiller — APPROVED** [review]

File: frontend/constants/serviceTemplates.ts (hunk 2 above)
Comment (Tanya-atatakai) [reply frontend/constants/serviceTemplates.ts:183]: we don't use these variables on the UI, we only plan to populate the values from this form, can remove both name 
 and description actually, if you don't use them
image

Comment (OjusWiZard) [reply frontend/constants/serviceTemplates.ts:183]: okay, then it should be fine to keep the description empty. But, the name was probably there to keep it as one source of truth, and then dynamically show that in the UI. But, that's FE discussion so probably @truemiller can comment on that.

On the BE side, it should be fine as long as the values are correctly populated for each KEY in the mapping

**Review by OjusWiZard — APPROVED** [review]

File: frontend/constants/serviceTemplates.ts [hunk 4]
```diff
@@ -157,31 +157,61 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
+      },
+      TWIKIT_PASSWORD: {
+        name: 'Twitter password',
+        description: '',
+        value: '',
+        provision_type: EnvProvisionType.USER,
+      },
+      GENAI_API_KEY: {
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply frontend/constants/serviceTemplates.ts:202]: I hope 😅 https://github.com/dvilelaf/meme-ooorr-quickstart/blob/main/run_service.py#L778

**Pull request merged**